src/answers-umd.js

```diff
@@ -509,9 +509,11 @@ class Answers {
    * @param {string} translations The translations, or a stringified JSON of possible translations
    * @param {Object} interpolationParams Params to use during interpolation
    * @param {number} count The count associated with the pluralization
+   * @returns {SafeString} The translation as a Handlebars SafeString
    */
   translateJS (translations, interpolationParams, count) {
-    return Translator.translate(translations, interpolationParams, count);
+    const translation = Translator.translate(translations, interpolationParams, count);
+    return this.renderer.SafeString(translation);
```

Comment:
If this helper is called in JS rather than Handlebars, does it still work as expected?

Reply:
Same question here! I was mostly wondering whether this would work properly for analytics events. For context components can report analytics with this.analyticsReporter for example https://github.com/yext/answers/blob/c9f8a6b4329223633adbf32d00edcb95e63a6db0/src/ui/components/questions/questionsubmissioncomponent.js#L248

Reply:
It does when called with JS, however the result is wrapped in a SafeString which is essentially an object that looks like this: {string: "Yext"}. I think I am going to remove this though because of Tom's suggestions below